morphir/runtime/src/org/finos/morphir/runtime/quick/Loop.scala

```diff
@@ -2,12 +2,11 @@ package org.finos.morphir.runtime.quick
 
 import org.finos.morphir.naming.*
```

Comment:
@DamianReeves Other than the one `Definition[UType, Unit]` case I commented on, this file is exactly what I was expecting to see. Can you take a look and make sure this is also what you were expecting/agreed to?